qutebrowser/config/configdata.yml

```diff
@@ -1091,6 +1091,14 @@ hints.mode:
       - word: Use hints words based on the html elements and the extra words.
   desc: Mode to use for hints.
 
+hints.extended_filters:
```

Comment:
This shouldn't be a boolean, but instead a list of strings (enum). That way, more types can be added later on.

(I still don't think elemstr should be modified in `_filter_matches`)

Reply:
Replaced with a list.